### PR TITLE
Data science content

### PR DESCRIPTION
**Pull request by shaloo**
Fixes content issue #73. First draft, please review and merge.

### CODE REVIEW

File: layouts/partials/tabs.html [hunk 1]
```diff
@@ -104,9 +104,81 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
...
+	<p>
+          Before machine learning algorithms can be applied to data, it requires
+	  cleaning and preparation by tools such as Pandas. <a href=
+          "https://seaborn.pydata.org">Seaborn</a> is well known for its <a href=
+	  "https://towardsdatascience.com/how-to-perform-exploratory-data-analysis-with-seaborn-97e3413e841d">
+	  exploratory data analysis (EDA)</a> capabilities, while some of the
+          backbone processes required for machine learning, namely, regression
+	  methods, classification, clustering, model validation and selection are
+	  well served by <a href="https://scikit-learn.org">Scikit-learn</a> and
+	  <a href="https://scikit-learn.org">Scipy</a> (statistical computing).
```
Comment (rgommers) [inline layouts/partials/tabs.html]: `scipy.org` rather than `scikit-learn.org` here

File: layouts/partials/tabs.html [hunk 2]
```diff
@@ -104,9 +104,81 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
...
+	</p>  
+        <p>
+	And of course, <b>NumPy</b> is the foundation of almost all of these tools
+	  used for Data Science.
+        </p>
+	<p>
+	Besides these, <a href="http://keras.io">Keras</a> offers deep learning
```
Comment (rgommers) [inline layouts/partials/tabs.html]: I would suggest to move this paragraph over to the Machine Learning tab - that'd be ML/AI.

Also, Keras is unfortunately end-of-life. It only supports Python 2.7-3.6, and development has stopped. Users should migrate to `tensorflow.keras`, so at this point Keras is no longer worth mentioning unfortunately.

Comment (rgommers) [reply layouts/partials/tabs.html]: Other suggestions for Machine Learning: 
- Visualization: Yellowbrick, Eli5
- ML libraries: XGBoost, LightGBM, CatBoost
- AI libraries: PyTorch, MXNet

Yellowbrick may have a nice visualization to steal, e.g. https://www.scikit-yb.org/en/latest/api/regressor/residuals.html#residuals-plot

File: layouts/partials/tabs.html [hunk 3]
```diff
@@ -104,9 +104,81 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
...
+	framework helps to define and run computations involving tensors that have
+	broad applications in Speech and image recognition, Text-based
+	applications, Time-series analysis and Video detection.
+	</p>
+	<div class="grid-container">
+            <div>
+	    <p>
+              Effective data analytics require deep knowledge of the data domain
+              (e.g., Retail, Healthcare, Marketing, Finance, Social Media,
+	      Automation, Sales, Travel, Pure Science, Aeronautics etc.) as well
```
Comment (rgommers) [inline layouts/partials/tabs.html]: I would stop after `Travel` and discard the rest of this paragraph, to keep the text to a reasonable length.

File: layouts/partials/tabs.html [hunk 4]
```diff
@@ -104,9 +104,81 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
...
+            </p>
+	    <p>
+              Python has a rich ecosytem of libraries that enable Data Science
+	      workflows. These workflows typically involve the following key
+	      processes:
+                <ul class="content-tab">
+                <li><b>Data Extraction:</b> Pandas, Beautiful Soup</li>
+                <li><b>Data Transformation:</b> Pandas</li>
+                <li><b>Data Manipulation:</b> Seaborn, Matplotlib</li>
+                <li><b>Data Exploration:</b> Scikit-learn, SciPy</li>
```
[the remark this answers is not in the document]
Comment (rgommers) [reply layouts/partials/tabs.html]: Also, I think the figure is quite nice. Isn't this list duplicate with the figure?

File: layouts/partials/tabs.html (hunk 3 above)
Comment (rgommers) [reply layouts/partials/tabs.html]: And then perhaps finish off with something like: "Effective Data Science workflows typically involve tools like MLflow for experiment hyperparameter and result tracking, and DVC for versioning data".

File: layouts/partials/tabs.html [hunk 5]
```diff
@@ -15,7 +15,7 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
         <nav class="cd-tabs__navigation">
         <ul class="cd-tabs__list">
             <li>
-            <a href="#tab-inbox" class="cd-tabs__item cd-tabs__item--selected">
+            <a href="#tab-visualisation" class="cd-tabs__item cd-tabs__item--selected">
```
[the remark this answers is not in the document]
Comment (shaloo) [reply layouts/partials/tabs.html]: The visualisation tab id was named tab-inbox and I renamed it to tab-visualisation so that it can be referred to in other tabs content (if required).  There are only two places where this id 'tab-inbox' was used in sources - both within the same file. No other content is referring to the id tab-inbox in html/md. For now, I will revert it back to the id name 'tab-inbox' and check out locally what's going on - at least I did not see tab content blanking out.

[the remark this answers is not in the document]
Comment (shaloo) [reply layouts/partials/tabs.html]: Ah! Got it - renaming to tab-vis is working now - no blanking out.  The link was opening reference in a new window - that wasn't needed. Thanks for catching this.

[the remark this answers is not in the document]
Comment (shaloo) [reply layouts/partials/tabs.html]: Fixed, tested https://github.com/numpy/numpy.org/commit/7f6ee54b18e82595dd448ec5b73f03f597e914c5

File: layouts/partials/tabs.html [hunk 6]
```diff
@@ -104,9 +104,81 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
...
+            <div>
+               <img src="images/content_images/ds-landscape.png" alt="ds-landscape" align="left">
+            </div>
+            <div>
+            <p>
+	      Data Science makes it possible to analyze massive amounts of data
+	      and gain meaningful insights. 
+            </p>
+	    <p>
+              Python has a rich ecosytem of libraries that enable Data Science
```
[the remark this answers is not in the document]
Comment (shaloo) [reply layouts/partials/tabs.html]: Fixed https://github.com/numpy/numpy.org/commit/0fed9865ed963d41885dbfab5ae7e346cb9009e8

File: layouts/partials/tabs.html [hunk 7]
```diff
@@ -104,9 +104,81 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
...
+              </ul>
+            </p>
+            </div>
+	</div>
+	<p>
+	  All data manipulation begins with gathering or extracting data. <a 
+	  href="https://pandas.pydata.org"> Pandas </a> helps in data discovery
+          and handling, while <a href=
+	  "https://www.crummy.com/software/BeautifulSoup/">Beautiful Soup</a>
+	  helps in gathering and web-scrapping data sets.
```
[the remark this answers is not in the document]
Comment (shaloo) [reply layouts/partials/tabs.html]: Fixed https://github.com/numpy/numpy.org/commit/0fed9865ed963d41885dbfab5ae7e346cb9009e8

File: layouts/partials/tabs.html (hunk 1 above)
Comment (shaloo) [reply layouts/partials/tabs.html]: Fixed. https://github.com/numpy/numpy.org/commit/0fed9865ed963d41885dbfab5ae7e346cb9009e8

File: layouts/partials/tabs.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (shaloo) [reply layouts/partials/tabs.html]: It isn't duplicate but I have condensed the content further based on your inputs and edited out the list.  Let me know if this suffices https://github.com/numpy/numpy.org/commit/3a9e5aa731474ec46d5ddc343a7c2822e3b478ad

File: layouts/partials/tabs.html (hunk 2 above)
Comment (shaloo) [reply layouts/partials/tabs.html]: Done - temporarily parked content in ML page - will edit that next - after DS is merged.

Comment (shaloo) [reply layouts/partials/tabs.html]: See https://github.com/numpy/numpy.org/commit/3a9e5aa731474ec46d5ddc343a7c2822e3b478ad

File: layouts/partials/tabs.html (hunk 3 above)
Comment (shaloo) [reply layouts/partials/tabs.html]: Included. Ref https://github.com/numpy/numpy.org/commit/9bbc977facda006ee1cfa3d166fd359be75c3979

File: layouts/partials/tabs.html [hunk 8]
```diff
@@ -104,9 +104,81 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
...
+	      and gain meaningful insights. 
+            </p>
+	    <p>
+              Python has a rich ecosytem of libraries that enable Data Science
+	      workflows. These workflows typically involve the following key
+	      processes:
+                <ul class="content-tab">
+                <li><b>Data Extraction:</b> Pandas, Beautiful Soup</li>
+                <li><b>Data Transformation:</b> Pandas</li>
+                <li><b>Data Manipulation:</b> Seaborn, Matplotlib</li>
```
[the remark this answers is not in the document]
Comment (shaloo) [reply layouts/partials/tabs.html]: Added.

**Comment by shaloo** [comment]
@rgommers: Is this ready for a merge to newsite branch?

**Comment by rgommers** [comment]
Yes, this looks good now, merged. Thanks @shaloo!

**Comment by joelachance** [comment]
Should we merge this into `master`?

**Comment by rgommers** [comment]
Oops, good catch. Synced to `master` in commit 8104f93. And I'll get rid of `newsite` so we won't have the same issue again.

**Comment by shaloo** [comment]
@joelachance, @rgommers , thx.
